CHANGES.md

```diff
@@ -62,8 +62,9 @@
 
 ## New Features / Improvements
 
-* X feature added (Java/Python) ([BEAM-X](https://issues.apache.org/jira/browse/BEAM-X)).
-
+*  Added support for downloading and staging wheels/bdists for the PyPI packages. Currently supported for `linux_86_64 platforms`.
```

Comment:
possible wording to consider: 

Pipeline dependencies supplied  through --requirements_file, will now be staged to the runner using their binary distributions (wheels) of the PyPI packages for `linux_x86_64 platform` ([BEAM-4032](https://issues.apache.org/jira/browse/BEAM-4032)). To restore the behavior to use source distributions, set pipeline option `--requirements_cache_only_sources`. To skip staging the packages at submission time, set pipeline option `--requirements_cache=skip`(Python)